src/components/sidebar/common/SectionResult.js

```diff
@@ -14,12 +15,23 @@ export const styles = {
 
 function SectionResult({ addSection, sectionHover, sectionHoverOff, section, disabled, classes }) {
   const sectionIsUnscheduled = section.schedules.some(isUnscheduled);
+  const { enqueueSnackbar } = useSnackbar();
+
+  const message = 'Successfully added';
```

Comment:
My personal preference, but it might be more descriptive to say something like 'Class successfully added'